src/Spd.Resource.Repository/User/OrgUserRepository.cs

```diff
@@ -247,18 +248,7 @@ private async Task<OrgUsersResult> SearchUsers(Guid? organizationId, Guid? ident
             if (identityId != null)
                 users = users.Where(a => a._spd_identityid_value == identityId);
 
-            var userList = users.ToList();

```

Comment:
Why did we have to remove this loop?

Reply:
no use, it will do it automatically in mapping